src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(){
+
+    }
```

Comment:
```suggestion
    public Truck() {
   
    }
```
make sure to auto format your files